### ISSUE TITLE
Run failure

### ISSUE THREAD

**Issue by Oliveche** [TropComplique/lda2vec-pytorch#4]
MY COMPUTER: NVIDIA CUDA 9.0.176 driver     Torch-0.3.0
When I run "train.py"  ， it always stops at the following interface.

Please help me , thank you very much.

**Comment by Oliveche** [comment]
MY COMPUTER: NVIDIA CUDA 9.0.176 driver Torch-0.3.0
When I run "train.py" ， it always stops at the following result：
number of documents: 12829
number of windows: 1023189
number of topics: 25
vocabulary size: 7460
word embedding dim: 50
number of batches: 143 

epoch 1
  0%|          | 0/143 [00:00<?, ?it/s]


Please help me , thank you very much.

**Comment by Oliveche** [comment]
?

**Comment by TropComplique** [comment]
Hi. Sorry, I can't help you with this. You need to do the debugging by yourself (I don't have enough information to help you).
